### PR TITLE
Add note to "help translate" page

### PR DESCRIPTION
**Pull request by WorldLanguages**
Adds a small note to the page. See diff

Reminder: also update TurboWarp's page accordingly.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
Also: should change "48 hours" to "96 hours"

File: content/docs/localization/joining-the-localization-team.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ Thanks for wanting to help with translating Scratch Addons to other languages! P
 
 If you don't own a Scratch account, please send an email to `translation@scratchaddons.com` instead.
 
+Do not create a new Scratch account just to fulfill this requirement. Instead, send an email to the address above. New scratcher accounts are ignored.
```
Comment (Hans5958) [inline content/docs/localization/joining-the-localization-team.md:69]:
```suggestion
**If you don't have a Scratch account**, please send an email to `translation@scratchaddons.com` instead. Do not create a new Scratch account just to fulfill this requirement. New Scratcher accounts will be ignored.
```

**Comment by Hans5958** [comment]
Also, can we remove line 60 and update line 13 to something like this?

> Thanks for the interest to help with translating Scratch Addons to other languages! Scratch Addons is available in more than 15 languages thanks to the help of volunteers around the world.
>
> Please read this page carefully to continue.

**Comment by WorldLanguages** [comment]
@Hans5958 Sure, go ahead and change it.

**Comment by WorldLanguages** [comment]
@Hans5958 Done

**Review by Hans5958 — APPROVED** [review]

**Comment by Hans5958** [comment]
You may merge at your own discretion, or I will do it tommorow.

**Pull request merged**